packages/lib/gpcircuits/src/util.ts

```diff
@@ -1,5 +1,6 @@
 import path from "path";
 import { CircuitArtifactPaths, CircuitDesc } from "./types";
+import * as fs from "fs/promises";
```

Comment:
This is also a Node module, and this util file is used in browsers, so I'm not sure if this will work.
Maybe the helpers you've put here belong in a shared file in the scripts directory, not the src directory?